sdk/eventhub/event-hubs/migrationguide.md

```diff
@@ -174,7 +181,7 @@ await subscription.close();
 See [`receiveEvents.ts`](https://github.com/Azure/azure-sdk-for-js/blob/master/sdk/eventhub/event-hubs/samples/typescript/src/receiveEvents.ts)
 for a sample program demonstrating this.
 
-### Migrating code from `EventHubClient` to `EventHubProducerClient` for sending events
+### Migrating from `EventHubClient` to `EventHubProducerClient` for sending events
```

Comment:
Here, it looks like creating a batch is the only option they have. Can we say that they can still send an array of events if they are sure that would fit in a batch and then if they want safe batching they can create a batch themselves? If doing so, add a note that the array option is only available from version 5.1.0 I think..

Reply:
Updated! (and it was version 5.2.0)